### PR TITLE
Add issue grab bag grant

### PR DESCRIPTION
**Pull request by vvisigoth**
Actually @jalehman can you take a quick look, I did 2 stars/3 issues because that seemed fair from the kind of issues that suggested themselves to me. Want to get your opinion.

### CODE REVIEW

File: content/grants/2023/issue-grab-bag.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
++++
+
+## Overview
+
+
+The Urbit Foundation monitors existing and new issues in the codebase to identify issues that can be addressed by community contributors. 
+
+This bounty will close any three of the issues labeled ["community contribution"](https://github.com/urbit/urbit/labels/community%20contribution) in the Urbit github repo.
+
+The ideal contributor will have a decent command of hoon and an understanding of the Urbit stack, but the issues are selected to minimize need for deep context.
```
Comment (jalehman) [inline content/grants/2023/issue-grab-bag.md]: Do any require runtime knowledge?

File: content/grants/2023/issue-grab-bag.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+work_request_link = "https://airtable.com/shr4qt9t9kz7RaOIa?prefillGrant+ID=&prefillGrant+Name=Urbit%20Issue%20Grab%20Bag"
+
++++
+
+## Overview
+
+
+The Urbit Foundation monitors existing and new issues in the codebase to identify issues that can be addressed by community contributors. 
+
+This bounty will close any three of the issues labeled ["community contribution"](https://github.com/urbit/urbit/labels/community%20contribution) in the Urbit github repo.
```
Comment (jalehman) [inline content/grants/2023/issue-grab-bag.md]: This reads strangely. The bounty closes issues? Or the contributor gets the bounty by resolving any three issues?

**Review by jalehman — COMMENTED** [review]
I'd expect that we can get at least two contributors doing this at a time. Probably only need one grant for that to start.

File: content/grants/2023/issue-grab-bag.md (hunk 1 above)
Comment (vvisigoth) [reply content/grants/2023/issue-grab-bag.md]: Clarified in https://github.com/urbit/urbit.org/pull/2020/commits/01b6d24d986993889196614a2f452a4bdf5d7b6f

File: content/grants/2023/issue-grab-bag.md (hunk 2 above)
Comment (vvisigoth) [reply content/grants/2023/issue-grab-bag.md]: Clarified in https://github.com/urbit/urbit.org/pull/2020/commits/01b6d24d986993889196614a2f452a4bdf5d7b6f

**Review by jalehman — APPROVED** [review]

**Pull request merged**